### PR TITLE
WIP: owner migration configuration

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2703** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2703" target="_blank">Graphite</a> 
    * **PR #2704** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2704" target="_blank">Graphite</a> 
      * **PR #2705** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2705" target="_blank">Graphite</a> 
        * **PR #2706** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2706" target="_blank">Graphite</a> 
          * **PR #2707** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2707" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2707?utm_source=stack-comment).

File: src/components/v2v3/V2V3Project/V2V3ProjectSettings/V2V3ProjectSettings.tsx [hunk 1]
```diff
@@ -164,6 +80,109 @@ export function V2V3ProjectSettings() {
     }
   }
 
+  const items = useMemo(() => {
+    const includeTokenMigration =
+      featureFlagEnabled(FEATURE_FLAGS.V1_TOKEN_SWAP) && cv !== '3'
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3ProjectSettings/V2V3ProjectSettings.tsx:85]: **suggestion** think theres a `CV_V3` constant or similar

File: src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/V1V2TokenMigrationSettingsPage/V1TokenMigrationSetupSection/V1TokenMigrationSetupModal/DeployMigrationTokenSection.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+              loading={addTerminalLoading}
+              disabled={
+                completed ||
+                userInputOccurring ||
+                !isValidV1ProjectId ||
+                addTerminalLoading ||
+                v1ProjectIdLoading
+              }
+            >
+              <Trans>Add terminal</Trans>
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/V1V2TokenMigrationSettingsPage/V1TokenMigrationSetupSection/V1TokenMigrationSetupModal/DeployMigrationTokenSection.tsx:216]:
```suggestion
              <Trans>Deploy migration token</Trans>
```

File: src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/V1V2TokenMigrationSettingsPage/V1TokenMigrationSetupSection/V1TokenMigrationSetupModal/V1TokenMigrationSetupModal.tsx [hunk 3]
```diff
@@ -1,74 +1,140 @@
...
+                className="bg-smoke-75 dark:bg-slate-400"
+                iconComponent={<CheckCircleFilled />}
+              >
+                <p>
+                  <Trans>You're all set!</Trans> 🎉
+                </p>
+                <p className="m-0">
+                  <Trans>
+                    V1 token holders can swap their tokens for your V2 tokens on
+                    your V2 project's <strong>Tokens</strong> section.
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/V1V2TokenMigrationSettingsPage/V1TokenMigrationSetupSection/V1TokenMigrationSetupModal/V1TokenMigrationSetupModal.tsx:130]:
```suggestion
                    V1 token holders can migrate their tokens to your V3 tokens using
                    your V3 project's <strong>Tokens</strong> section.
```

**Comment by wraeth-eth** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: iRcirx3Jve5L39rUJXmk.
See details on [graphite.dev](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2707)

**Pull request merged**